docs/tutorial/part06.rst

```diff
@@ -2,5 +2,440 @@
 How to serialize the data.
 ==========================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+Why we need to serialize the data
+=================================
+
+The nodes that we are fetching so far are just instances of Python objects, we want to return JSON files to the fetch calls that are going to be made in the frontend. Python objects structure is not JSON serializable and the nodes don't support the already build in serializers on the library, so we have to create our own serialize methods and manage the logic of those since sometimes we would need to serialize only the directly related nodes to a specific node, to serialize the data is nothing more than getting it from the object fromat to a parseable format.

```

Comment:
I think we should refine this paragraph.

Some key ideas: 
- The nodes that we are fetching are represented with instances of the model classes(Entity, Officer, etc).
- We want to serialize(translate node objects) into JSON format. 
- JSON data can be easily passed and consumed by the client(the browser).
- Adding a serialize method to the model classes is a convenient way to serialize the nodes objects, because each node will know how to translate itself...

Please refer to this definition of serialization https://en.wikipedia.org/wiki/Serialization